lib/foreman_scap_client/client.rb

```diff
@@ -190,5 +192,14 @@ def ensure_tailoring_file
     def download_uri(download_path)
       foreman_proxy_uri + "#{download_path}"
     end
+
+   def foreman_upload_result(response)
+     parsed = JSON.parse(response.body)
```

Comment:
what if proxy runs older version of smart_proxy_openscap and does not response with valid JSON? should we rescue the parsing error? I'm getting following error on such scenario

```
Uploading results to https://foreman.example.tst:8443/compliance/arf/5
Upload failed: A JSON text must at least contain two octets!
```